packages/modular-scripts/src/analyze.ts

```diff
@@ -0,0 +1,8 @@
+import actionPreflightCheck from './utils/actionPreflightCheck';
+import { getPackageDependencies } from './utils/getPackageDependencies';
+
+async function analyze({ target }: { target: string }): Promise<void> {
+  console.log(JSON.stringify(await getPackageDependencies(target), null, 2));
```

Comment:
Can we wrap this with `strip-ansi` to strip escape characters? This will be important for cross platform users